src/test/ui/editions/edition-keywords-2015-2018-expansion.stderr

```diff
@@ -7,8 +7,8 @@ LL |     produces_async! {}
    = note: this error originates in a macro (in Nightly builds, run with -Z macro-backtrace for more info)
 help: you can escape reserved keywords to use them as identifiers
    |
-LL | () => (pub fn r#async () { })
-   |               ^^^^^^^
+LL |     () => (pub fn r#async() {})
+   |                   ^^^^^^^
```

Comment:
So it shouldn't block this PR, but I just realized that nowhere does it actually show the *file* for this suggestion (it's in the JSON AFAIK, but that's because the JSON diagnostics are much more uniform).

Do non-suggestion "child diagnostics" get to print their own file/line/column?

cc @rust-lang/wg-diagnostics

Reply:
Elsewhere in the diff there's this, mid-diagnostic, to "switch files":
```diff
-  ::: <::ping::ping macros>:1:1
+  ::: $DIR/auxiliary/ping.rs:5:1
```

So it is plausible that suggestions could also check if they need to show this.